packages/contracts/contracts/L1/Colosseum.sol

```diff
@@ -882,6 +897,19 @@ contract Colosseum is Initializable, Semver {
         return _turn % 2 == 0;
     }
 
+    /**
+     * @notice Determines whether the given status is ready to be proven.
+     *
+     * @param _status Current status of the challenge.
+     *
+     * @return Whether the given status is ready to be proven.
+     */
+    function _isReadyToProve(ChallengeStatus _status) internal view returns (bool) {
```

Comment:
Why not `private pure`?

Reply:
This function is only used in one place, so it will be removed.